python/src/LeastSquaresExpansion_doc.i.in

```diff
@@ -0,0 +1,62 @@
+%feature("docstring") OT::LeastSquaresExpansion
+"L2 approximation on an orthonormal basis using least-squares and a fixed basis.
+
+Available constructors:
+    LeastSquaresExpansion(*inputSample, outputSample, distribution*)
+
+    LeastSquaresExpansion(*inputSample, weights, outputSample, distribution*)
+
+    LeastSquaresExpansion(*inputSample, outputSample, distribution, basis, basisSize, methodName*)
+
+    LeastSquaresExpansion(*inputSample, weights, outputSample, distribution, basis, basisSize, methodName*)
+
+Parameters
+----------
+inputSample : 2-d sequence of float of size :math:`n>0`
+    Input sample.
+weight : sequence of float of size :math:`n>0`, optional
+    Input weights. If not given, the default value is a uniform weight equals to :math:`1/n`.
+outputSample : 2-d sequence of floats of size :math:`n>0`
+    Output sample.
+distribution : :class:`~openturns.Distribution`
+    The distribution associated to the weighted sample *(inputSample, weights)*
+basis : :class:`~openturns.OrthogonalBasis`, optional
+    The orthonormal basis of the vector space on which the approximation is built.
+    If not given, a basis is built using an hyperbolic enumerate function
+    :class:`~openturns.HyperbolicAnisotropicEnumerateFunction` associated to the
+    :math:`q-`norm specified by the 'FunctionalChaosAlgorithm-QNorm' key in
+    :class:`~openturns.ResourceMap`.
+    The basis is obtained through a tensorization of the univariate orthonormal
+    polynomial bases associated to each marginal distributions of *distribution*.
+basisSize : int, optional
+    The size of the basis on which the approximation is built. If not given, it is
+    given by the 'FunctionalChaosAlgorithm-BasisSize' key in
+    :class:`~openturns.ResourceMap` if this value is positive, or as the cardinal
+    of the set of basis functions up to a degree given by the
+    'FunctionalChaosAlgorithm-MaximumTotalDegree' key in
+    :class:`~openturns.ResourceMap`.
+methodName : str, optional
+    The decomposition method used to solve the least-squares problem, of
```

Comment:
```suggestion
    The decomposition method used to solve the least-squares problem, 
```

Reply:
`of type` is used in `KrigingResult_doc.i.in`, `LeastSquaresStrategy_doc.i.in`, `RandomVectorImplementation_doc.i.in`, `SpecFunc_doc.i.in` and `TaylorExpansionMoments_doc.i.in`. Should I fix them in the same shot?
BTW it looks weird to me to read `the least-squares problem, type :class:...`